### PR TITLE
static folder, context processor, layout

### PR DESCRIPTION
**Pull request by sylvesteralex**

### CODE REVIEW

**Comment by ko0owi** [comment]
......

**Review by Jordan4321 — APPROVED** [review]
ok

File: cars/models.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ class Car(models.Model):
 
     brand = models.CharField(max_length= 32)
     model = models.CharField(max_length= 32)
-    image = models.ImageField(upload_to='media/cars', default='static/img/placeholder.png')
+    image = models.ImageField(upload_to='cars/', blank=True)
```
Comment (Jordan4321) [inline cars/models.py:35]: ok

File: cars/templates/cars/details.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
     {% include 'navbar.html' %}
 
-    <section class="hero is-medium is-link">
+    <section class="hero hero-subsite is-link">
```
Comment (Jordan4321) [inline cars/templates/cars/details.html:12]: ok

File: cars/templates/cars/details.html [hunk 3]
```diff
@@ -35,10 +35,15 @@ <h2 class="title is-2">{{ car.brand }} {{ car.model }}</h2>
                             <div class="columns">
                                 <div class="column">
                                     <h3 class="title is-3">Details</h3>
+                                    <span class="material-symbols-outlined">settings_suggest</span>
```
Comment (Jordan4321) [inline cars/templates/cars/details.html:38]: ok

**Review by Jordan4321 — APPROVED** [review]

File: website/views.py [hunk 4]
```diff
@@ -37,4 +37,29 @@ def login(request):
...
+
+    context = {
+        'page_title': "Terms and conditions",
+    }
+
+    return render(
+        request,
+        'website/terms.html',
+        context=context
+    )
```
Comment (Jordan4321) [inline website/views.py]: ok

**Review by Jordan4321 — APPROVED** [review]

**Pull request merged**